pkg/ingester/instance.go

```diff
@@ -84,7 +84,7 @@ func (i *instance) Push(ctx context.Context, req *logproto.PushRequest) error {
 			i.streamsCreatedTotal.Inc()
 		}
 
-		if err := stream.Push(ctx, s.Entries); err != nil {
+		if err := stream.Push(ctx, s.Entries, blockSize); err != nil {
```

Comment:
Instead of sending the blockSize in with every Push, we can simplify this.

We can make it a member of `instance` and `stream`, and pass it in here: https://github.com/grafana/loki/blob/b85609e5ef9f287646a5c136e2205abdf1e0c678/pkg/ingester/ingester.go#L157 and here: https://github.com/grafana/loki/blob/b85609e5ef9f287646a5c136e2205abdf1e0c678/pkg/ingester/stream.go#L61

Reply:
Hey sorry for delay reply, this is a great way i want too, i will change it.